tests/test_parsing.py

```diff
@@ -23,7 +23,7 @@ def _to_test_data(text):
 # ========= adjectives
 хорошему            хороший     ADJF,Qual masc,sing,datv
 лучший              хороший     ADJF,Supr,Qual masc,sing,nomn
-наиневероятнейший   вероятный   ADJF,Supr,Qual masc,sing,nomn
```

Comment:
You can use [these changes](https://github.com/insolor/pymorphy3/commit/958b4dc5dd33e5c21b4c936024e0c813cef23c10#diff-0c76c033dbce347029f7291ebb3db59b16c982432f9ccabb0b308a0c2e1383d8) to test_parsing and use XFAIL instead of changing to an other word

Reply:
done in https://github.com/no-plagiarism/pymorphy3/pull/9/commits/b673d50855ca2a4ee0c8d861994490dbdb4c6448